test/e2e/framework/framework.go

```diff
@@ -266,6 +266,13 @@ func (f *Framework) AfterEach() {
 		LogPodsWithLabels(f.Client, api.NamespaceSystem, ImagePullerLabels)
 	}
 
+	// Dump the output of the nethealth containers only once per run
+	if TestContext.DumpNethealthLogs {
+		By("Dumping network health container logs from all nodes")
+		LogContainersInPodsWithLabels(f.Client, api.NamespaceSystem, ImagePullerLabels, "nethealth")
+		TestContext.DumpNethealthLogs = false
```

Comment:
Maybe just do this in https://github.com/kubernetes/kubernetes/blob/master/test/e2e/e2e.go#L145, inconditionally, if you only want it once?